packages/gatsby-mdx/gatsby/extend-node-type.js

```diff
@@ -194,119 +194,68 @@ ${code}`;
 
     return resolve({
       code: {
```

Comment:
The changes to the `code` field make it easier to gather the scopes inside the Webpack loaders and make it possible to pull in the exact scope needed in the `MDXRenderer` instead of all of the scopes, as it does now.

  **Current:**

  The `code` field is a GraphQL Object (meaning each of these fields must be individually queried) that contains the following children:

  * `raw` - The raw MDX code
  * `body` - The compiled code ready to be rendered by the `MDXRenderer`
  * `scope` - A file path to a file which exports all the dependencies for the MDX scope

  **New:**

  The `code` field is a GraphQL JSON object (meaning these fields are queried for implicitly) that contains the following children:

  * `_mdxCode` - A boolean that is always set to `true`. This makes it easier for us to find MDX scopes when we run the GraphQL queries in the webpack loaders.
  * `body` - The compiled code ready to be rendered by the `MDXRenderer`
  * `scope` - An object with the `id` to use for the imported scope, and the `location` of the file which exports all the dependencies for the MDX scope